spec/foodzone/venues/list_retriever_spec.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../../../app/foodzone/venues/list_retriever'
+require_relative '../../../app/foodzone/venues/address'
+require_relative '../../../app/foodzone/venues/telephone'
+
+describe Venues::ListRetriever do
+  let(:retriever) { described_class }
+  let(:venue_repo) { double :venue_repo, all: venues }
+
+  context '::execute' do
+    context 'for empty list of venues' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.